CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # CHANGELOG
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [0.11.2] - 2023-8-16
+- Added Zifencei, Bit Manipulation and Privilege tests ctg files for RV32E
```

Comment:
you can either avoid this change or rebase to the dev and move this line under the `WIP-DEV` section of the CHANGELOG